### DISCUSSION TITLE
How to load glb files using BabylonNative in iOS Swift ?

### DISCUSSION THREAD

**Discussion by MouGABSI** [BabylonJS/BabylonNative#1171] [Development]
I would like to load glb files using BabylonNative but i doesn't find any documentation in your repository.
Is there a way to add BabylonNative as a dependency using any dependency package Manager (cocoapods or other)
how can i load glb content using this libarary ?
It is strange as library where can i find documentation ?

**Comment by bghgary** [comment]
Please ask questions on our [forum](https://forum.babylonjs.com). Thanks!
